shell/src/main/java/alluxio/cli/fs/command/StatCommand.java

```diff
@@ -94,7 +101,15 @@ protected void runPlainPath(AlluxioURI path, CommandLine cl)
   @Override
   public int run(CommandLine cl) throws AlluxioException, IOException {
     String[] args = cl.getArgs();
-    AlluxioURI path = new AlluxioURI(args[0]);
+    AlluxioURI path;
+    if (cl.hasOption("fileId")) {
+      try (CloseableResource<FileSystemMasterClient> client =
+          mFsContext.acquireMasterClientResource()) {
+        path = new AlluxioURI(client.get().getFilePath(Long.parseLong(args[0])));
```

Comment:
Print a message that says `The specified file ID 123 is /path/to/file` here.

Reply:
done